bundles/org.openhab.binding.shelly/src/main/feature/feature.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.shelly-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${project.version}/xml/features</repository>
+
+    <feature name="openhab-binding-shelly" description="Shelly Binding" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
+        <feature>openhab-transport-mdns</feature>
+        <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.shelly/${project.version}</bundle>
```

Comment:
I prepared a draft for the update to Californium 2.0.0 in OHC (https://github.com/openhab/openhab-core/pull/1201). After it will have been merged we can use `<feature>openhab.tp-coap</feature>` and remove the `*.jar`-files from this PR.

Reply:
ok

Reply:
done, core feature openhab.tp-coap is referenced and Californium libs are removed from bundle (lib folder)